core/src/main/scala/com/salesforce/op/evaluators/OpRegressionEvaluator.scala

```diff
@@ -55,6 +60,50 @@ private[op] class OpRegressionEvaluator
 
   @transient private lazy val log = LoggerFactory.getLogger(this.getClass)
 
+  final val signedPercentageErrorHistogramBins = new DoubleArrayParam(
+    parent = this,
+    name = "signedPercentageErrorHistogramBins",
+    doc = "sequence of bins for signed percentage error histogram",
```

Comment:
is this the error percentage bins or the prediction value bins? lets make the doc clear